components/bpmn/org.wso2.carbon.bpmn.ui/src/main/resources/web/bpmn/process_definition.jsp

```diff
@@ -92,7 +94,8 @@
                 </tr>
                 <tr>
                     <td><fmt:message key="bpmn.process.deployedDate"/></td>
-                    <td><%=process.getDeploymentTime().toString()%>
+                    <% DateFormat dateOutputFormatter = new SimpleDateFormat("MM/dd/yyyy"); %>
```

Comment:
we cannot define static final variables in JSP. what we can do here is, define it in a Class and reference that class from JSPs. Rather than creating a new java class to hold only this (to keep date format), i think it should be better to keep in the same jsp file.